src/SourceBuild/Arcade/eng/common/templates/job/source-build-build-tarball.yml

```
@@ -68,7 +68,7 @@ jobs:
   - ${{ if ne(parameters.excludeSdkContentTests, 'true') }}:
     - download: ${{ parameters.installerBuildResourceId }}
       artifact: BlobArtifacts
-      patterns: '**/dotnet-sdk-+([0-9]).+([0-9]).+([0-9])?(-preview*|-rc*)-linux-${{ parameters.architecture }}.tar.gz'
+      patterns: '**/dotnet-sdk-+([0-9]).+([0-9]).+([0-9])?(-preview*|-rc*|-rtm)-linux-${{ parameters.architecture }}.tar.gz'
```

```suggestion
      patterns: '**/dotnet-sdk-+([0-9]).+([0-9]).+([0-9])?(-preview*|-rc*|-rtm*)-linux-${{ parameters.architecture }}.tar.gz'
```
Do you need the wildcard here or are you sure it will work without it?

---

I debated if it was needed and explicitly decided against it because I'm only aware of a single rtm release.